vars/CentOS_9.yml

```diff
@@ -0,0 +1,8 @@
+---
+# This system supports drop in directory so defaults are adjusted
+__ssh_supports_drop_in: true
+__ssh_drop_in_name: "00-ansible"
+
+# This default lists the main configuration file defaults
+__ssh_defaults:
+  Include: /etc/ssh/ssh_config.d/*.conf
```

Comment:
Please, do not add copies of clones  as long as they are same as the originals. Use symlinks as with the other vars files.

Reply:
Fixed